apps/linode-marketplace-nats-single-node/roles/nats/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+
+- name: install required packages
```

Comment:
Can we simplify this into something like  (I'm nit picking here, but just less lines for same result :))

`- name: install packages
  apt:
    pkg:
    - net-tools
    - curl
    - unzip
    state: present`

Reply:
Done!